### ISSUE TITLE
NTR: 2-oxoisocaproic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8578]
New term request 2-oxoisocaproic acid high in urine (segal_030509170445): 2-oxoisocaproate; alpha-ketoisocaproic acid; 4-methyl-2-oxopentanoic acid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000599
- label: Elevated urinary 2-oxoisocaproic level
- parents: [Increased urine proteinogenic amino acid derivative level (HP:0033097)](https://hpo.jax.org/app/browse/term/HP:0033097)
- pmid: 
- def: The amount of 2-oxoisocaproic in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Elevated levels of 2-oxoisocaproic acid can be found in individuals with Maple syrup urine disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8578
- synonyms:
